cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -187,6 +187,17 @@ func generateFileContent(gen *protogen.Plugin, file *protogen.File, g *protogen.
 	}
 }
 
+// Copy comments from proto file to _grpc.pb.go file.
+func copyCommentsFromProto(g *protogen.GeneratedFile, service *protogen.Service) {
+
+	// Pint comment without '\n', if comment has '\n', godoc considers this annotation to be non-service annotation.
```

Comment:
the inline comments in this function can be removed

Reply:
Instead can be more explicit in the docstring. Something like

`// copyCommentsFromProto copies the comments from the RPC proto definitions`
`// to the corresponding generated interface file.`